db/migrate/20240202064506_create_regions.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRegions < ActiveRecord::Migration[7.1]
+  def change
+    create_table :regions, id: :uuid do |t|
+      t.string :name
+      t.decimal :hourly_rate, precision: 10, scale: 2
```

Comment:
This attribute name is a bit too generic. Maybe `funding_available_per_hour` based on the document below:

![CleanShot 2024-02-02 at 11 04 12](https://github.com/DFE-Digital/itt-mentor-services/assets/4231530/8107cf1a-6f1d-4b3f-a6d0-c0db5d12c83b)

Reply:
Also, as it's a a claims service only attribute, maybe we should prefix it? `claims_funding_available_per_hour`?

The alternative would be to create a `Claims::Region` that points to the base `Region` with additional attributes on the `Claims::Region` model/table, but it feels like a bit too much for now.

Reply:
Also, I've always found a better experience working with money through https://github.com/RubyMoney/money-rails. What do you think?

Might be a question for everyone: @DFE-Digital/school-placements @DFE-Digital/track-pay

Reply:
I prefer to work with the money gem, make is much easier and don't have to think about rounding issues 😄  👍🏻